### DISCUSSION TITLE
react-email should be added as dev dependency

### DISCUSSION THREAD

**Discussion by angelhodar** [resend/react-email#1575] [General]
I recently installed react-email in my nodejs backend to send my emails (its working perfectly!!) but I noticed that my backend docker image went from 500MB to 1GB, so I noticed its because the docs says to install the `react-email` package that contains the nextjs preview app as a normal dependency instead of a dev one. Using qnm, you can see that it adds a lot of dependencies regarding nextjs like swc:

```
size    count module
------ -----  ----------
137.63mb 1 ✓  @next/swc-linux-x64-musl
122.13mb 1 ✓  googleapis
114.62mb 1 ✓  @next/swc-linux-x64-gnu 
84.71mb  1 ✓  next
54.16mb  1 ✓  @swc/core-linux-x64-musl
45.06mb  1 ✓  @swc/core-linux-x64-gnu 
38.3mb   1 ✓  typescript
10.27mb  1 ✓  @firebase/database      
9.29mb   1 ✓  esbuild
9.16mb   1 ✓  @esbuild/linux-x64
8.04mb   1 ✓  libphonenumber-js
6.25mb   1 ✓  @react-email/tailwind
```
Compared to the base one when moving `react-email` to `devDependencies`:

```
size    count module
------ -----  ----------
122.13mb 1 ✓  googleapis
10.27mb  1 ✓  @firebase/database     
8.04mb   1 ✓  libphonenumber-js      
6.25mb   1 ✓  @react-email/tailwind  
5.28mb   1 ✓  @google-cloud/firestore
5.1mb    1 ✓  class-validator        
4.61mb   1 ✓  stripe
4.32mb   1 ✓  react-dom
```

Maybe I am missing something but docs should differentiate between components and render packages from the preview server one

**Comment by gabrielmfern** [comment]
You are completely right on this, but #1537 should already help with the issue of bundle size — as it will have Next and other dependencies as devDependencies. And one more thing, as more of a heads-up, in the future, we will probably merge all components into the `react-email` package to improve the developer experience a bit more. So, having it as a devDependency won't be an option in the future.

But currently we do have plans to guide users into installing `react-email` as a dev dependency as well #1522.

**Reply by angelhodar** [reply]
Oh sorry I didnt find that PRs, i searched in discussions and issues and I thought no one commented about it, my bad. If you are already aware of the bundle issue and have that PR to fix it, then I have nothing more to say! Thanks for your time
